checksums.bzl

```diff
@@ -3,54 +3,66 @@
 # TO GENERATE THIS RUN: ./updateWorkspaceSnapshots.sh
 
 BASE_ARCHITECTURES = ["amd64", "arm64"]
-ARCHITECTURES = BASE_ARCHITECTURES + ["s390x"]
+ARCHITECTURES = BASE_ARCHITECTURES + ["s390x", "ppc64le"]
```

Comment:
Why can't this be in `BASE_ARCHITECTURES`?

Reply:
@chanseokoh i was just following the pattern set by the s390x :) we can revisit this in a subsequent PR.

Reply:
I think that like `s390x` this is a starting point to get base & static in place?

Reply:
I believe the only reason that we distinguish `BASE_ARCHITECTURES` and `ARCHITECTURES` is to include or exclude `libunwind8` because Debian doesn't have the library package on s390x.

So, the consequence of this would be that, Distroless won't be able to build ppc64le dotnet images (dotnet requires `libunwind8`, as it seems).

Reply:
Not that this is a major issue. We just need to be clear on why we have both `BASE_ARCHITECTURES` and `ARCHITECTURES` for what reason.

Reply:
Yeah, but `s390x` doesn't publish python, node, or other images yet, so practically I think the "base" signifies what we publish `//base` for, but this may be back-filling the naming a bit 😅

Reply:
Yeah, but I do think `ppc64le` (basically any arch that doesn't need special handling such as x390x) should be in `BASE_ARCHITECTURES`. I can tolerate a brief transition state if this is to be fixed in a subsequent PR.